core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/udf/TestCase.java

```diff
@@ -0,0 +1,60 @@
+package edu.uci.ics.texera.dataflow.udf;
```

Comment:
The test case should go in the tests folder. The main portion here can be encapsulated inside a @Test function. The table creation portion can go in the @setup portion.